src/components/SearchForm/Holiday/Holiday.js

```diff
@@ -36,12 +34,7 @@ class Holiday extends Component {
       };
 
       const createName = (className) => {
-        return ( this.state.sign )
```

Comment:
explain the reason of this formatting.
first of all - right now at search api we still not finished our main method that will handle all cases with this form submission. so i decided to use included/excluded term because i think it'll be more easy to work from server side.
But! it's still not clear turnaround for me, so i decided to format like this and have ability to apply and easy changes, because this format is really eye catchy